dotnet/Datadog.AzureAppServices.DotNet.nuspec

```diff
@@ -1,17 +1,17 @@
 <?xml version="1.0"?>
 <package>
   <metadata>
-    <id>Datadog.Development.AzureAppServices</id>
-    <version>100.5.15-prerelease</version>
-    <title>Very Experimental .NET Datadog APM</title>
+    <id>Datadog.AzureAppServices.DotNet</id>
+    <version>1.0.0</version>
+    <title>.NET Datadog</title>
```

Comment:
Should the title include "APM" or "Tracer" anywhere?

Reply:
I thought not, as this is ultimately what they use within AAS to enable every aspect of Datadog they can for .NET applications.
I intentionally excluded those terms.
Do you disagree?

Reply:
I agree with Lucas, I think we should keep it as .NET Datadog APM. Sure, it does more, but that's the primary function I'd say since it's a serverless offering